Problem5/src/problem5/NewJFrame.java

```diff
@@ -0,0 +1,380 @@
+            .addComponent(jPanel1, javax.swing.GroupLayout.PREFERRED_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.PREFERRED_SIZE)
+        );
+
+        pack();
+    }// </editor-fold>//GEN-END:initComponents
+
+    private void jButton1ActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_jButton1ActionPerformed
+
+        try {
+            File file = new File("results.txt");
+
+            /*
+            Checking if the file results.txt exixts and 
+            if it doesn't create the file
+            */
+            
+            if (!file.exists()) {
+                file.createNewFile();
+            }
+
+            /*
+            Checking the options the user chose and adding it to the variable data
+            */
+            
+            if (shooters.isSelected()) {
+                sh++;
+            }
+            if (fighters.isSelected()) {
+                fi++;
+            }
+            if (platformers.isSelected()) {
+                pl++;
+            }
+            if (strategy.isSelected()) {
+                st++;
+            }
+            if (rpg.isSelected()) {
+                rp++;
+            }
+            if (puzzle.isSelected()) {
+                pu++;
+            }
+            if (arcade.isSelected()) {
+                ar++;
+            }
+            if (sports.isSelected()) {
+                sp++;
+            }
+            if (other.isSelected()) {
+                ot++;
+            }
+            String time = hours.getSelectedItem().toString();
+            if ("< 5 hours".equals(time)) {
+                h5++;
+            }
+            if ("< 10 hours".equals(time)) {
+                h10++;
+            }
+            if ("< 15 hours".equals(time)) {
+                h15++;
+            }
+            if ("15+ hours".equals(time)) {
+                h15plus++;
+            }
+            favgames += games.getText();
+            favgames += "\n";
+            submits++;
+            
+            /*
+            Calling the method that updates the file
+            */
+            
+            updateFile();
+            
+            /*
+            Setting the program to it's starting appearance
+            */
+            
+            shooters.setSelected(false);
+            fighters.setSelected(false);
+            platformers.setSelected(false);
+            strategy.setSelected(false);
+            rpg.setSelected(false);
+            puzzle.setSelected(false);
+            arcade.setSelected(false);
+            sports.setSelected(false);
+            other.setSelected(false);
+            hours.setSelectedIndex(0);
+            games.setText("");
+
+        } catch (IOException e) {
+            System.out.println(e);
+        }
+
+
+    }//GEN-LAST:event_jButton1ActionPerformed
+
+        /**
+         *
+         * @throws IOException
+         */
+        public void updateFile() throws IOException {
```

Comment:
You should be following OOP best practices.  You should have created a helper class to handle updating the results file with new survey data.  The GUI code and the program logic should be split into separate classes whenever possible.  It is a good practice because if the GUI needs to change (a frequent occurrence for various reasons), it would probably have little to no effect on the rest of the program logic.  If you wanted to add unit tests for your program logic, splitting the code up would make it easier to do so.